templates/security/notices.html

```diff
@@ -1 +1,142 @@
-{% extends "templates/one-column.html" %}
\ No newline at end of file
+{% extends "templates/one-column.html" %}
+{% set search = request.args.get('release') or request.args.get('details') %}
+{% block title %}Security notices{% endblock %}
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-12">
+      {% if search %}
+      <h2>Ubuntu Security Notices - Search Results</h2>
```

Comment:
This should also be a `h1`:
```suggestion
      <h1>Ubuntu Security Notices - Search Results</h1>
```

Reply:
I think we want it as an `h2`
https://github.com/canonical-web-and-design/ubuntu.com/pull/6420#issuecomment-584080664.

Reply:
In that case, `<h1 class="p-heading--two">`, the page should have a `h1`.